db/re/job_exp.yml

```diff
@@ -0,0 +1,3286 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2021 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Job Experience Group Database
+###########################################################################
+#
+# Job Experience Group Settings
+#
+###########################################################################
+# - ExpGroup                 Experience group ID.
+#   Jobs:                    Jobs associate to experience group.
+#     Job                    Job associated to experience group.
+#   MaxBaseLevel             Maximum base level. (Default: MAX_LEVEL)
+#   BaseExp:                 Base experience per level.
+#     - Level                Base level.
+#       Exp                  Base experience.
+#   MaxJobLevel              Maximum job level. (Default: MAX_LEVEL)
+#   JobExp:                  Job experience per level.
+#     - Level                Job level.
+#       Exp                  Job experience.
+###########################################################################
+
+Header:
+  Type: JOB_EXP
+  Version: 1
+
+Body:
+  - ExpGroup: 1
```

Comment:
How about removing the unused ExpGroup field?
The user just need to specify the jobs on import as key
```
  - Jobs:
      Novice: true
      Swordman: true
# fields the user want to update for these jobs
```

Reply:
This unfortunately is needed to help separate each normal grouping. If we were to do by Job then our table would grow exponentially with duplicate data.